core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/externaldatabase/ExternalDatabaseService.java

```diff
@@ -67,7 +68,15 @@ public void provisionDatabase(Cluster cluster, DatabaseAvailabilityType external
         AllocateDatabaseServerV4Request request = getDatabaseRequest(environment, externalDatabase, cluster);
         String databaseCrn;
```

Comment:
this should be inside the try. also would be nicer if the `if` part would be in a separate method and there would be only one value assignment for this variable

Reply:
not possible to move the variable declaration into the `try` block as the `waitAndGetDatabase` needs it too and that is outside the try `block`